src/coreclr/jit/llvm.h

```diff
@@ -108,6 +108,7 @@ class Llvm
     std::unordered_map<GenTree*, Value*>* _sdsuMap;
     std::unordered_map<SsaPair, Value*, SsaPairHash>* _localsMap;
     std::vector<PhiPair> _phiPairs;
+    std::vector<Value*>* m_allocas;
```

Comment:
`std::vector`s, AFAIK, are fairly small in size in most standard libraries, so I don't think there's a need for additional indirections.
```suggestion
    std::vector<Value*> m_allocas;
```
(Will also entail changing code which uses this)